types/sanctuary/index.d.ts

```diff
@@ -125,9 +169,9 @@ declare namespace Sanctuary {
     clamp(a: string): (b: string) => (c: string) => string;
     clamp(a: string[]): (b: string[]) => (c: string[]) => string[];
     clamp(a: StrMap<string>): (b: StrMap<string>) => (c: StrMap<string>) => StrMap<string>;
```

Comment:
I believe definitions for arrays of primitives should be removed, since Ord type already covers it (hopefully)